packages/analytics-browser/src/config/joined-config.ts

```diff
@@ -31,6 +32,9 @@ export class BrowserJoinedConfigGenerator {
       this.config.defaultTracking = remoteConfig.defaultTracking;
     }
     this.config.loggerProvider.debug('Joined configuration: ', JSON.stringify(remoteConfig, null, 2));
+    if (this.remoteConfigFetch && this.remoteConfigFetch.fetchTime) {
+      this.config.requestMetadata = new RequestMetadata(this.remoteConfigFetch.fetchTime);
```

Comment:
Could this potentially remove other existing metadata? Should you check for existing first? e.g.

```
this.config.requestMetadata = this.config.requestMetadata ?:new RequestMetadata();
this.config.requestMetadata.fetchTime = this.remoteConfigFetch.fetchTime
```

Reply:
I did it because fetchTime is the only meta data right now. But it make sense to be more scalable. I set only fetch time at https://github.com/amplitude/Amplitude-TypeScript/pull/772/commits/0be281dd9a063ad66fb6e15ded979b34cbfc2fdb